tools/docker/Dockerfile.proto-builder

```diff
@@ -0,0 +1,14 @@
+FROM golang:1.12-stretch
```

Comment:
This file needs the license header.

Reply:
Went ahead and added it to tools/docker/Dockerfile.runc-builder to.